test/site/_collections/posts-with-lighthouse/dev.spec.js

```diff
@@ -0,0 +1,23 @@
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+const outputPath = path.join(
+  __dirname,
+  '..',
+  '.tmp',
+  path.basename(__dirname),
+  'pages',
+  'collection',
+  'index.html',
+);
+
+describe('posts-with-lighthouse', function() {
+  describe('postsWithLighthouse', function() {
+    it('creates postsWithLighthouse collection in dev env', async function() {
```

Comment:
I think the message may be redundant because mocha will be showing:

```
_collections
  in DEV env
    posts-with-lighthouse
      creates postsWithLighthouse collection in dev env
```

One question to ask: what aspect of this output do we care about? Did it include some files and exclude others? Maybe we would say that it output the expected number of files with the `pathItem` tag and the `web_lighthouse` field?

You could also add test fixtures that have `web_lighthouse: N/A` and maybe junk stuff like `web_lighthouse: true` or null, etc.